### DISCUSSION TITLE
Chatgroup Permission Negation

### DISCUSSION THREAD

**Discussion by BirminghamBoy** [Zrips/CMI#2331] [Q&A]
**Question: **
I've set up numerous player ranks with different chat groups and also have set up staff with their own chat group. Since I want my staff to play on the server, they inherit the player permissions as well as their own. However, my staff's chat is in the player format, rather than the staff format (which I set their permissions for). I've tried negating the player permissions from them, but it's not working. Any ideas?

**Comment by kassadGLA** [comment]
`cmi.chatgroup.[groupnumber]`
Your staff group must have `[groupnumber]` > then regular players group. Did you do that?
Example:
regular player group - `cmi.chatgroup.1`
staff group - `cmi.chatgroup.4`

**Comment by BirminghamBoy** [comment]
> `cmi.chatgroup.[groupnumber]`
> Your staff group must have `[groupnumber]` > then regular players group. Did you do that?
> Example:
> regular player group - `cmi.chatgroup.1`
> staff group - `cmi.chatgroup.4`

No, I didn't. Works now, thanks!

**Comment by CloudeLecaw** [comment]
would be nice if you close the ticket ;)
